crates/client/rpc/src/lib.rs

```diff
@@ -977,6 +977,53 @@ where
         Ok(estimates)
     }
 
+    /// Estimate the L2 fee of a message sent on L1
+    ///
+    /// # Arguments
+    ///
+    /// * `message` - starknet transaction request
```

Comment:
"starknet transaction request" that doesn't make a lot of sense to me.
Wouldn't we better with something along the lines of "the message to estimate"